src/shared/services/api-config.service.ts

```diff
@@ -17,8 +17,30 @@ export class ApiConfigService {
     return this.nodeEnv === 'production';
   }
 
-  private getNumber(key: string): number {
-    return Number(this.configService.get(key));
+  private getNumber(key: string, defaultValue?: number): number {
+    const value = this.configService.get(key, defaultValue);
+    if (value === undefined) {
+      console.error(key + ' env var not set');
+      throw new NotFoundException(key + ' env var not set');
```

Comment:
@tolstenko please change exception to Internal server error, so Back-end dev/Devops will fix this error in any case

Reply:
No problem. thanks

Reply:
probably we should call process.exit() too to avoid locking the service

Reply:
done